mira/metamodel/ops.py

```diff
@@ -51,6 +52,11 @@ def stratify(
         The (singular) name of the stratification, e.g., ``"city"``
     strata :
         A list of the values for stratification, e.g., ``["boston", "nyc"]``
+        or ``[geonames:4930956, geonames:5128581]``.
+    strata_name_map :
```

Comment:
let's rename this to `strata_curie_to_name` so it's explicit what is in this dictionary

Reply:
Resolved in 3950e7fb5ff69005213a56d4541a8c07a17c81ee